### PR TITLE
Aea 02

### PR DESCRIPTION
**Pull request by AEArntsen**
mainly just bug fixes after playing with different data

### CODE REVIEW

File: TACT/computation/.#adjustments.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+XSym
```
Comment (nrgpy) [inline TACT/computation/.#adjustments.py]: @AEArntsen can you please remove this from the repo?

Comment (nrgpy) [reply TACT/computation/.#adjustments.py]: also, remove the two `.DS_store` files?

Comment (AEArntsen) [reply TACT/computation/.#adjustments.py]: all set

**Review by dave-c-vt — APPROVED** [review]
test pasess with --timetestFlag disabled.

**Pull request merged**